src/orchard/apropos.clj

```diff
@@ -17,28 +17,44 @@
 
 ;;; ## Symbol Search
 
-(defn- apropos-sort
+(defn- safe-comparator [x y]
+  (compare (pr-str x) (pr-str y)))
+
+(defn- default-comparator [ns clojure-ns?]
+  (fn [x y]
+    (cond
+      (= x y)  0
```

Comment:
Added a `(= x y)  0` making the sort stable.

Unfortunately I couldn't prove this fixes the issue, however it's most likely an improvement.